parser/src/token.rs

```diff
@@ -364,6 +366,20 @@ impl<'input> Tokenizer<'input> {
         self.error(start, UnterminatedStringLiteral)
     }
 
+    fn shebang_line(&mut self, start: Location) -> Option<SpannedToken<'input>> {
+
+        let (end, line) = self.take_until(start, |ch| ch == '\n');
+        
+        if line.starts_with("#!") {
+            let skip = 2;
+            let result = line[skip..].trim_right().to_string();
```

Comment:
If you remove the `to_string` the you will get an `&'input str instead` (or omit it, see above)